controller/tasks_manager.go

```diff
@@ -122,11 +122,11 @@ func (tm *TasksManager) TaskCreateAndRun(ctx context.Context, taskConfig config.
 func (tm *TasksManager) TaskDelete(ctx context.Context, name string) error {
 	logger := tm.logger.With(taskNameLogKey, name)
 	if tm.drivers.IsMarkedForDeletion(name) {
-		logger.Debug("task is already marked for deletion")
+		logger.Debug("task is already marked for deletion", "taskName", name)
```

Comment:
This actually has the task name as part of the logging already, as set on line 123.

```
tm.logger.With(taskNameLogKey, name)
```

```
2022-06-13T09:12:21.691-0500 [DEBUG] tasksmanager: task marked for deletion: task_name=test-task
2022-06-13T09:12:21.691-0500 [DEBUG] tasksmanager: task deleted: task_name=test-task
```